src/components/WalletConnect/NanoContract/NewNanoContractTransactionModal.js

```diff
@@ -42,10 +39,8 @@ export const NewNanoContractTransactionModal = ({
   };
 
   useEffect(() => {
-    if (isRetrying) {
```

Comment:
The code is working but maybe we could add a config here to skip dApp request warning screen, then it could be merged to `master` as well

Reply:
That would be great, refactored it, can you check again?